lib/src/room.dart

```diff
@@ -1844,22 +1844,16 @@ class Room {
         50;
   }
 
-  bool get canCreateGroupCall =>
-      canChangeStateEvent(EventTypes.GroupCallPrefix) && groupCallsEnabled;
-
   bool get canJoinGroupCall =>
-      canChangeStateEvent(EventTypes.GroupCallMemberPrefix) &&
-      groupCallsEnabled;
+      canChangeStateEvent(VoIPEventTypes.FamedlyCallMemberEvent);
 
   /// if returned value is not null `org.matrix.msc3401.call.member` is present
   /// and group calls can be used
   bool get groupCallsEnabled {
     final powerLevelMap = getState(EventTypes.RoomPowerLevels)?.content;
     if (powerLevelMap == null) return false;
-    return powerForChangingStateEvent(EventTypes.GroupCallMemberPrefix) <=
-            getDefaultPowerLevel(powerLevelMap) &&
-        powerForChangingStateEvent(EventTypes.GroupCallPrefix) <=
-            getDefaultPowerLevel(powerLevelMap);
+    return powerForChangingStateEvent(VoIPEventTypes.FamedlyCallMemberEvent) <=
+        getDefaultPowerLevel(powerLevelMap);
   }
```

Comment:
Isn't this the same as canChangeStateEvent?

Reply:
yes

Reply:
actually no, getDefaultPowerLevel checks if group calls are enabled for all other users with default power